src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -138,48 +146,75 @@ const CardListItemIndex = ({
     };
     const enrollments = item.tei ? item.tei.enrollments : [];
     const enrollmentType = deriveEnrollmentType(enrollments, currentProgramId);
-    const { orgUnitName, enrolledAt, programFromEnrollment, programNameFromEnrollment } = deriveEnrollmentOrgUnitAndDate(enrollments, enrollmentType, currentProgramId);
+    const { orgUnitName, enrolledAt } = deriveEnrollmentOrgUnitAndDate(enrollments, enrollmentType, currentProgramId);
+    const program = enrollments && deriveProgramFromEnrollment(enrollments, currentSearchScopeType);
 
+    const renderTag = () => {
+        switch (enrollmentType) {
+        case enrollmentTypes.ACTIVE:
+            return (
+                <Tag
+                    dataTest="dhis2-uicore-tag"
+                    positive
+                    icon={
+                        <span className={classes.checkIcon}>
+                            <IconCheckmark16 />
+                        </span>
+                    }
+                >
+                    {i18n.t('Enrolled')}
+                </Tag>
+            );
+        case enrollmentTypes.CANCELLED:
+        case enrollmentTypes.COMPLETED:
+            return (
+                <Tag
+                    dataTest="dhis2-uicore-tag"
+                    neutral
+                    icon={
+                        <span className={classes.checkIcon}>
+                            <IconCheckmark16 />
+                        </span>
+                    }
+                >
+                    {i18n.t('Previously enrolled')}
+                </Tag>
+            );
+        default:
+            return null;
+        }
+    };
+    console.log(program);
```

Comment:
Should remove these 2 `console` lines before merging the PR. Besides this everything looks good!